tests/basic_one_way_send.go

```diff
@@ -0,0 +1,99 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/teleporter/tests/utils"
+	. "github.com/ava-labs/teleporter/tests/utils"
+
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+// Ginkgo describe node that acts as a container for the teleporter e2e tests. This test suite
+// will run through the following steps in order:
+// 1. Send a transaction to the Teleporter contract on Subnet A
+// 2. Aggregate signatures and send the Warp message to Subnet B
+// 3. Verify receipt of the message on Subnet B
+func BasicOneWaySend() {
+	var (
+		teleporterMessageID *big.Int
+	)
+
+	subnetAInfo := utils.GetSubnetATestInfo()
+	subnetBInfo := utils.GetSubnetBTestInfo()
+	teleporterContractAddress := utils.GetTeleporterContractAddress()
+	fundedAddress, fundedKey := utils.GetFundedAccountInfo()
+
+	//
+	// Send a transaction to Subnet A to issue a Warp Message from the Teleporter contract to Subnet B
+	//
+	ctx := context.Background()
+
+	nonceA, err := subnetAInfo.ChainWSClient.NonceAt(ctx, fundedAddress, nil)
```

Comment:
With multiple `It`s running tests in parallel, I think we may have a race condition around the nonce usage here. 

Similarly, we could have race conditions around waiting for a transaction to be included in a block, because there may be a new block produced after a subscription is opened but before the transaction is sent/accepted by the network, given that other goroutines are also sending other transactions at the same time.

Reply:
Oh for sure. To be clear, the transition from having tests in a `Describe` vs a `It` container buys us the following:
- More idiomatic usage of Ginkgo
- Unordered test runs (good for fuzzing)

It does NOT buy us the ability to run the tests in parallel on its own. As you mentioned, there are concurrency issues with nonces, block channels, etc. We have to be careful when setting up the global vs thread state to account for this. This [draft PR](https://github.com/ava-labs/teleporter/pull/34) goes into more detail on some of those points.

Reply:
Gotcha, I forgot that the specs as is will still run serially, just not in a specific order.